src/random.cpp

```diff
@@ -170,6 +189,26 @@ static uint64_t GetRdSeed() noexcept
         __asm__ volatile ("pause");
     } while(true);
     return r1;
+#elif defined(_MSC_VER) && defined(_M_X64)
+    //TODO: Make this actually work
```

Comment:
This doesn't seem like a good TODO to leave :smile:

Reply:
OMG! I must be going blind from staring at code too long. How did I forget to delete that comment?
As penance, let me write some unittests specifically targeting `GetRdSeed` and `GetRdRand`.